mozci/queries/push_tasks_from_unittest.query

```diff
@@ -2,6 +2,9 @@ from: unittest
 select:
     - {name: id, value: task.id}
     - {name: groups, value: run.suite.groups}
+    - {name: result_group, value: result.group}
+    - {name: result_test, value: result.test}
+    - {name: result_ok, value: result.ok}
```

Comment:
I'm a bit confused by what's happening here. Doesn't a single task have hundreds of results? Is this actually going to select the right thing?

Reply:
Right, we should turn it into a list when we build the task

Reply:
It is now a list of Results.